lib/rules/web/admin_console/4.5/operator_hub.xyaml

```diff
@@ -266,3 +266,9 @@ open_operator_modal:
     keyword: <operator_name>
   action: filter_by_keyword_on_catalog_page
   action: open_first_card_in_overlay
+click_checkbox_from_providertype:
+  params:
+    filter_type_value: providerType
+  action: wait_filter_section_loaded
+  action: choose_one_checkbox
+
```

Comment:
There is an existing rule: click_checkbox_from_provider_type, could you check if it's the same function with "click_checkbox_from_providertype"

Reply:
What's the change for ?